src/test/java/pl/edu/zut/mad/schedule/search/MessageProviderSearchTest.kt

```diff
@@ -0,0 +1,28 @@
+package pl.edu.zut.mad.schedule.search
+
+import okhttp3.ResponseBody
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.Test
+import pl.edu.zut.mad.schedule.R
+import retrofit2.HttpException
+import retrofit2.Response
+
+class MessageProviderSearchTest {
+
+    private val messageProvider = MessageProviderSearch()
+
+    @Test
+    fun providerReturnIdResEmptyScheduleAlbumNumberNotFoundWhenErrorIs404() {
```

Comment:
Maybe use ``? It is a little bit har to read

Reply:
I mention in last PR that there is weird issue with using backticks, I will try to investigate this.